emba

```diff
@@ -725,6 +738,26 @@ main() {
     trap 'initial_status_bar' WINCH
   fi
 
+  #######################################################################################
+  # Quests (Q-modules)
+  #######################################################################################
+  if [[ "${CONTAINER_NUMBER}" -eq 2 ]] ;  then
+    while ! grep -q "Pre-checking phase started" "$LOG_DIR"/"$MAIN_LOG_FILE"; do
+      sleep 1
+    done
+    run_modules "Q" "${THREADED}" "0"
+    [[ ${THREADED} -eq 1 ]] && wait_for_pid "${WAIT_PIDS[@]}"
```

Comment:
Please double quote every variable access ... we try to do this consistent without any possibility for errors